src/components/BaseButton/index.tsx

```diff
@@ -1,18 +1,25 @@
 import React, { ReactNode } from 'react'
 import styled from '@emotion/styled'
-import { space, color, fontWeight, radius, device } from '../../theme'
+import { space, color, fontWeight, radius, device, fontSize } from '../../theme'
 import { baseTextStyles } from '../Text'
+import { css } from '@emotion/react'
 
 export enum ButtonVariant {
   success = 'success',
   warning = 'warning',
   danger = 'danger',
 }
 
+export enum BaseButtonSize {
+  medium = 'medium',
```

Comment:
Was thinking if we should call this `default` instead 🤔

Reply:
Hmm yes that is also a good option. I was trying to keep it inline with the normal button though, maybe there are other people with opinions about it? 😄

Reply:
I'm all for keeping it inline with the normal button